statusHandler/termuic/termuiRenders/widgetsRender.go

```diff
@@ -0,0 +1,217 @@
+	currentRound := currentRoundI.(int)
+
+	isSyncingI, _ := wr.termuiConsoleMetrics.Load(core.MetricIsSyncing)
+	isSyncing := isSyncingI.(int)
+
+	wr.prepareSyncInfoForDisplay(nonce, currentRound, synchronizedRound, isSyncing)
+
+	publicKeyI, _ := wr.termuiConsoleMetrics.Load(core.MetricPublicKey)
+	publicKey := publicKeyI.(string)
+	wr.preparePublicKeyForDisplay(publicKey)
+
+	shardIdI, _ := wr.termuiConsoleMetrics.Load(core.MetricShardId)
+	shardId := shardIdI.(int)
+	wr.prepareShardIdForDisplay(shardId)
+
+	numConnectedPeersI, _ := wr.termuiConsoleMetrics.Load(core.MetricNumConnectedPeers)
+	numConnectedPeers := numConnectedPeersI.(int)
+
+	txPoolLoadI, _ := wr.termuiConsoleMetrics.Load(core.MetricTxPoolLoad)
+	txPoolLoad := txPoolLoadI.(int)
+	wr.prepareTxPoolLoadForDisplay(txPoolLoad)
+	wr.prepareNumConnectedPeersForDisplay(numConnectedPeers)
+
+	countConsensusI, _ := wr.termuiConsoleMetrics.Load(core.MetricCountConsensus)
+	countConsensus := countConsensusI.(int)
+
+	countLeaderI, _ := wr.termuiConsoleMetrics.Load(core.MetricCountLeader)
+	countLeader := countLeaderI.(int)
+
+	countAcceptedBlocksI, _ := wr.termuiConsoleMetrics.Load(core.MetricCountAcceptedBlocks)
+	countAcceptedBlocks := countAcceptedBlocksI.(int)
+
+	wr.prepareConsensusInformationsForDisplay(countConsensus, countLeader, countAcceptedBlocks)
+
+	wr.prepareListWithLogsForDisplay(logLines)
+
+	return
+}
+
+func (wr *WidgetsRender) prepareListWithLogsForDisplay(logData []string) {
+	wr.lLog.Title = "Log info"
+	wr.lLog.Rows = wr.prepareLogLines(logData)
+	wr.lLog.TextStyle = ui.NewStyle(ui.ColorYellow)
+	wr.lLog.WrapText = false
+
+	return
+}
+
+func (wr *WidgetsRender) preparePublicKeyForDisplay(publicKey string) {
+	wr.pPublicKey.Title = fmt.Sprintf("PublicKey")
+	wr.pPublicKey.Text = fmt.Sprintf(publicKey)
+
+	return
+}
+
+func (wr *WidgetsRender) prepareShardIdForDisplay(shardId int) {
+	//Check if node is in meta chain
+	//Node is in meta chain if shardId is equals with max uint32 value
+	if shardId == int(^uint32(0)) {
+		wr.pShardId.Text = "ShardId: meta"
+	} else {
+		wr.pShardId.Text = fmt.Sprintf("ShardId: %v", shardId)
+	}
+
+	return
+}
+
+func (wr *WidgetsRender) prepareTxPoolLoadForDisplay(txPoolLoad int) {
+	wr.pTxPoolLoad.Text = fmt.Sprintf("Tx pool load: %v", txPoolLoad)
+
+	return
+}
+
+func (wr *WidgetsRender) prepareNumConnectedPeersForDisplay(numConnectedPeers int) {
+	wr.pNumConnectedPeers.Text = fmt.Sprintf("Num connected peers: %v", numConnectedPeers)
+
+	return
+}
+
+func (wr *WidgetsRender) prepareConsensusInformationsForDisplay(countConsensus int, countLeader int, acceptedBlocks int) {
+	wr.pCountConsensus.Text = fmt.Sprintf("Count consensus group: %v", countConsensus)
+	wr.pCountLeader.Text = fmt.Sprintf("Count leader: %v", countLeader)
+	wr.pCountAcceptedBlocks.Text = fmt.Sprintf("Accepted blocks: %v", acceptedBlocks)
+
+	return
+}
+
+func (wr *WidgetsRender) prepareSyncInfoForDisplay(nonce, currentRound, synchronizedRound, isSyncing int) {
```

Comment:
bad parameter name. When I see isSyncing I automatically think that the parameter is of type bool.

Reply:
name of variable was changed with syncStatus.